README.md

```diff
@@ -86,3 +86,7 @@ biggest changes planned so far.
 ## License
 
 Headlamp is released under the terms of the [Apache 2.0](./LICENSE) license.
+
+## Frequently Asked Questions
+
+For more information about Headlamp, see the [Headlamp FAQ](https://www.headlamp.dev/docs/latest/faq).
```

Comment:
Actually, the URL should not have the page prefix, so they are also usable in Github. i.e. [Headlamp FAQ](/docs/latest/faq). Same for all places where the relative path will work both in the browser and in Github.

Reply:
done!